tests/5.0/target/test_target_if_device.c

```diff
@@ -0,0 +1,58 @@
+//===------ test_target_if_device.c --------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the target directive with parallel for + if + device clauses
+//
+//===------------------------------------------------------------------------===//
+#include <stdio.h>
+#include "omp.h"
+#include "ompvv.h"
+#define THREADS 1024*1024*1024
```

Comment:
```suggestion
#define N 1024
```
This define shouldn't be named THREADS since it is not the number of threads. And I think it should be that large, since we are interested on functionality no stress testing